css-view-transitions-1/Overview.bs

```diff
@@ -1210,7 +1210,7 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 				1. Set |height| to the current height of |capturedElement|'s [=new element=]'s [=border box=].
 
-				1. Set |transform| to a transform that maps the |capturedElement|'s [=new element=]'s [=border box=] from document origin to its quad from the [=snapshot viewport origin=].
+				1. Set |transform| to a transform that maps the [=snapshot root origin=] to the top-left corner of the |capturedElement|'s [=new element=]'s [=border box=].
```

Comment:
I'm trying to think of clearer ways to word this. What about:

```
1. Set |transform| to a transform that would place |capturedElement|'s [=new element=]'s [=border box=] in its current visual position, from the [=snapshot root origin=].
```

Although I'm happy to go with what you have here.